sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/OtherTests.java

```diff
@@ -96,6 +98,38 @@ private DBSPCompiler compileDef() {
         return compiler;
     }
 
+    // Test using the JIT executor as a service.
```

Comment:
Why is this a service? It looks like the cargo command to run the JIT starts a process that reads inputs and then terminates with an exit code?

Reply:
That process is a service.
Previously we used to link the test in the same executable.
Once this works we'll use it for all our JIT tests.